skyvern/services/script_service.py

```
@@ -1798,7 +1866,119 @@ async def prompt(
+                organization_id=organization_id,
+            )
+            return
+        else:
+            await loop_block.build_block_result(
+                success=False,
+                failure_reason="No iterable value found for the loop block",
+                status=BlockStatus.terminated,
+                workflow_run_block_id=workflow_run_block_id,
+                organization_id=organization_id,
+            )
+            raise Exception("No iterable value found for the loop block")
```

Ruff (0.13.1)

Create your own exception

(TRY002)